File: Two Sum/Two_Sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        HashMap<Integer, Integer> mpp = new HashMap<>();
        int n = nums.length;
        for (int i = 0; i < n; i++) {
            int remaining = target - nums[i];
            if (mpp.containsKey(remaining)) {
                return new int[] {i, mpp.get(remaining)};
            }
            mpp.put(nums[i], i);
        }
        // dummy statement
        return new int[] {-1};
    }
}

File: Two Sum/Two_Sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        mpp = {}
        for i, num in enumerate(nums):
            remaining = target - num
            if remaining in mpp:
                return [i, mpp[remaining]]
            mpp[num] = i
        # dummy statement
        return [-1]
        